api/photos/models/photos.settings.json

```diff
@@ -12,30 +12,25 @@
   },
   "pluginOptions": {},
   "attributes": {
-    "uploaderName": {
-      "type": "string"
-    },
     "description": {
       "type": "richtext"
     },
     "photo": {
       "collection": "file",
```

Comment:
Not sure what this field means, but I noticed this was changed to 'model' in other scheme files